.flake8

```diff
@@ -37,9 +37,8 @@ per-file-ignores =
   #     F811 redefinition of unused '...'
   stdlib/typing.pyi: B, E701, E741, F401, F403, F405, F811
   # Generated protobuf files include docstrings
-  # *_pb2.pyi: B, E701, E741, F401, F403, F405, Y021, Y026, Y053, Y054
+  # NQA102: Flake8-PYI monkey-patches F821 in a way it no longer triggers on class def
```

Comment:
We should add a comment here saying that NQA102 should be removed from this list when we upgrade to `mypy-protobuf==3.5.0` and regenerate the stubs.

Although, since `mypy-protobuf==3.5.0` is already out, maybe we can just ask one of the maintainers on a Unix machine to regenerate the mypy-protobuf stubs for us using `mypy-protobuf==3.5.0`? Then we don't have to have any TODO comments anymore...

@JelleZijlstra / @srittau / @hauntsaninja? :)

I believe all that's required is to bump the pinned version here and then re-run the script:

https://github.com/python/typeshed/blob/608eca5234d92ef62a308365fed49b14aa4db748/scripts/generate_proto_stubs.sh#L16

Reply:
on it

Reply:
I ran it with 3.5.0 and didn't get any changes in the generated files relative to master. Are you sure mypy-protobuf 3.5.0 fixes the lint issue? I don't see anything in the above discussion that suggests that's the case.

Reply:
Also, the script didn't quite work for me on both macOS and Linux, probably due to minor differences in how commands like `sed` work. It's hard to write portable shell scripts.

Reply:
> I ran it with 3.5.0 and didn't get any changes in the generated files relative to master. Are you sure mypy-protobuf 3.5.0 fixes the lint issue?

Huh. I'm not 100% sure. I assumed it was fixed in 3.5.0 due to https://github.com/nipunn1313/mypy-protobuf/pull/525#issuecomment-1656478258, which I think predates the 3.5.0 release

Reply:
Whether it does fix it or not, do we want to bump anyway? Although it might be good to be able to run the script first ^^"

Reply:
I can submit a PR with a version that works for me on Linux, but not sure it will be any more portable than what's currently there.

Reply:
Wait I was wrong, I forgot to actually bump the version before I started running it on Linux. In fact the new version does remove the noqa comments.

Reply:
With #10914 completed, I'm marking this conversation as resolved.